CONTRIBUTING.md

```diff
@@ -182,3 +182,12 @@ Git tag for the full version number, like `git tag 0.21.1`, and
 push the tag to GitHub.
 
 After the release, you need to cherry-pick the release notes prep from `stable/*` to the `main` branch _and_ the stable branch for the latest minor version. For example, if the latest minor version is `0.3` and you released the patch `0.2.3`, then you need the copy the release notes `0.2.3.rst` into both `main` and `stable/0.3`. This ensures that the release notes show up for all future versions.
+
+### Cheatsheet for release process
+
+* Create branch ``jt-release-notes-x.x.x`` (ie: ``jt-release-notes-0.9.1``)
+* Update [docs/conf.py](docs/conf.py) and [setup.py](setup.py) with new x.x.x version
+* Run ``towncrier build --version=x.x.x --yes`` so release notes from [release-notes/unreleased](release-notes/unreleased) are flatten to [release-notes/x.x.x.rst](release-notes/x.x.x.rst)
```

Comment:
Should we explain how to add the info to `release-notes/unreleased`?

Reply:
Those unreleased release notes should be already included from previous pull requests, but I can indicate here that we must review all the included changes just in case we forgot to add the release notes on their PR

Thanks!

Reply:
thank you!

Reply:
https://github.com/Qiskit/qiskit-ibm-transpiler/pull/127/commits/7dbbbf713c7d1bad0132563a66409dc912b62f63